modules/platforms/dotnet/pom.xml

```diff
@@ -0,0 +1,113 @@
+        <artifactId>ignite-parent-internal</artifactId>
+        <version>${revision}</version>
+        <relativePath>../../../parent-internal/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>ignite-donet-thin-client</artifactId>
+    <packaging>pom</packaging>
+
+    <properties>
+        <ignite.thin.full.version>
+            ${ignite.majorVersion}.${ignite.minorVersion}.${ignite.incrementalVersion}.${ignite.buildNumber}
+        </ignite.thin.full.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>update-versions</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>initialize</phase>
+                        <configuration>
+                            <target>
+                                <echo message="Update ignite.version in .NET" />
+
+                                <replaceregexp byline="true" encoding="UTF-8">
+                                    <regexp pattern="(\[assembly:\s*Assembly\w*Version\w*\(&quot;)\d+\.\d+\.\d+(\.\d+)?(&quot;\)\])" />
+                                    <substitution expression="\1${ignite.thin.full.version}\3" />
+                                    <fileset dir="${basedir}/">
+                                        <include name="**/SharedAssemblyInfo.cs" />
+                                    </fileset>
+                                </replaceregexp>
+
+                                <replaceregexp byline="true" encoding="UTF-8">
+                                    <regexp pattern="(\[assembly:\s*AssemblyInformationalVersion\w*\(&quot;).*?(&quot;\)\])" />
+                                    <substitution
+                                        expression="\1${ignite.majorVersion}.${ignite.minorVersion}.${ignite.incrementalVersion}\2" />
+                                    <fileset dir="${basedir}/">
+                                        <include name="**/SharedAssemblyInfo.cs" />
+                                    </fileset>
+                                </replaceregexp>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>build-dotnet</id>
+                        <phase>generate-sources</phase>
+                        <configuration>
+                            <tasks>
+                                <exec executable="pwsh" dir="${project.basedir}" failonerror="true">
+                                    <arg line="${project.basedir}/build.ps1"/>
+                                    <arg line="-skipJava -skipDotNet -skipDotNetCore -jarDirs ${project.basedir}/bin/libs"/>
```

Comment:
Why `skipDotNet` and `skipDotNetCore`? We should build both of them.

I believe it should be `-skipJava -skipExamples`. Because Java is already built, and examples are published as source code.

Reply:
Thank you for you comment.
I've updated the pom.xml and also the DEVNOTES according to your suggestion.